puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_katello_devel_box.yaml

```diff
@@ -0,0 +1,11 @@
+- job:
+    name: test_katello_devel_box
+    node: 'admin && sshkey'
+    scm:
+      - forklift
+    triggers:
+      - github
+      - pollscm:
```

Comment:
I'm not sure polling the SCM makes sense here. I think this should be an ordinary cron since there are many moving pieces (packages, installer, forklift).